frame/oracle/src/lib.rs

```diff
@@ -739,6 +757,23 @@ pub mod pallet {
 			}
 		}
 
+		fn possible_reward(standard_reward: BalanceOf<T>) -> BalanceOf<T> {
+			let max_reward_per_block = Self::max_reward_per_block();
+			sp_std::cmp::min(standard_reward, max_reward_per_block)
+		}
+
+		pub fn max_reward_per_block() -> BalanceOf<T> {
+			let total_issuance = <T as Config>::Currency::total_issuance();
```

Comment:
we have rewards in LBP/vesting/crowdloan. it seems usually it tracks previos reward and ups it more inf next step is more than gap step from previous. so not sure what goal it here, may be design should be different

Reply:
may you leave comment in code? 
as i got it is some safety measure, so it would be clear from code.